core/domain/exp_services.py

```diff
@@ -807,6 +807,9 @@ def _save_exploration(committer_id, exploration, commit_message, change_list):
 
     exploration.version += 1
 
+    # Update summary of changed exploration.
```

Comment:
Yes, but could you make those tests use update_exploration() instead? On the whole, it'd be better if the tests didn't call private methods.